README.md

```diff
@@ -29,13 +29,45 @@ Run:
 
 ## <a name="contributing"></a>Contributing
 
- 1. **Fork** the repo on GitHub
- 2. **Clone** the project to your own machine
- 3. **Commit** changes to your own branch
- 4. **Push** your work back up to your fork
- 5. Submit a **Pull request**
+ 1. **[Fork](https://help.github.com/articles/fork-a-repo/#fork-an-example-repository)** the repo on GitHub
+ 2. **[Clone](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork)** the project to your own machine. Replacing YOUR-USERNAME with your github username.
+   ```bash
+    $ git clone https://github.com/YOUR-USERNAME/recordexpungPDX.git
+   ```
+ 3. cd into recordexpungPDX
+   ```bash
+    $ cd recordexpungPDX
+   ```
+ 4. Configure upstream to sync with your fork
+   ```bash
+    $ git remote add upstream https://github.com/CodeForPortland/recordexpungPDX.git
+   ```
+ 5. Create a branch to work on. Replacing BRANCH_NAME with a descriptive name of the work planned such as `update_contributing_doc`
+   ```bash
+     $ git checkout -b BRANCH_NAME
+   ```
+ 6. **Commit** changes to your branch (never to master)
```

Comment:
nit: this is not a hard and fast rule but one can have it as a convention. When you push to master, as long as the default upstream is your fork, it should be fine.

Reply:
ah I see. That is a good point.

Reply:
A suggestion for a resource if you find video's useful [Egghead.io - Contributing to OpenSource](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github). 
-YouTube Academy Alumni